crates/uv-cli/src/lib.rs

```diff
@@ -3121,7 +3121,7 @@ pub struct ToolUninstallArgs {
 pub struct ToolUpgradeArgs {
     /// The name of the tool to upgrade.
     #[arg(required = true)]
-    pub name: Option<PackageName>,
+    pub name: Option<Vec<PackageName>>,
```

Comment:
Can these be `Vec<PackageName>`? I think there's no semantic difference between `None` and `Some(vec![])` in this case.

Reply:
that's true, I've changed it to `Vec<PackageName>`

Reply:
Can we do the same for uninstall?

Reply:
Created a follow up PR: https://github.com/astral-sh/uv/pull/7077